testing/cockroach/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Joe Holden <jwh@zorins.us>
+# Maintainer: Joe Holden <jwh@zorins.us>
+pkgname=cockroach
+pkgver=2.1.5
+_pkgver=v2.1.5
+pkgrel=1
+pkgdesc="The open source, cloud-native SQL database."
+url="https://cockroachlabs.com"
+arch="x86_64"
+license="APL2"
```

Comment:
I am guessing the license is Apache-2.0. Please use the SPDX naming scheme https://spdx.org/licenses/

Reply:
Yup, changed